protobufs/livekit_sip.proto

```diff
@@ -154,7 +160,8 @@ message SIPTrunkInfo {
   // User-defined metadata for the Trunk.
   string metadata = 12;
 
-  // NEXT ID: 15
+  SIPMediaEncryption media_encryption = 15;
```

Comment:
Isn't SIPTrunkInfo deprecated? If so, do we want to add new options to it?

Reply:
Oops! Good catch! Fixed.